src/policy/fees.cpp

```diff
@@ -390,8 +390,9 @@ void CBlockPolicyEstimator::processTransaction(const CTxMemPoolEntry& entry, boo
         mapMemPoolTxs[hash].bucketIndex = feeStats.NewTx(txHeight, (double)feeRate.GetFeePerK());
     }
     else {
-        LogPrint("estimatefee", "not adding\n");
+        LogPrint("estimatefee", "not adding");
     }
+    LogPrint("estimatefee", "\n");
```

Comment:
I apologize for the ugly code again, but in the `if` and `else if`, `feeStats` and `priStats` both call `NewTx` respectively.  `NewTx` adds a new line.   I'd suggest leaving your change, but removing the new line in the `LogPrint` call in `NewTx`.  Or, removing all the `LogPrint` lines altogether, what are your thoughts about the level of debugging?  I found this useful in writing this code and for certain modifications it would be also valuable, but perhaps doesn't have much everyday value.  I'm running into the same questions in the mempool limiting code where debugging the various failures is valuable when working on the code but perhaps not when its finished until someone wants to change it.